### ISSUE TITLE
Invalid check_id provided

### ISSUE THREAD

**Issue by christianledgard** [cqchecker/codequiry-python#1]
Hello!

I'm receiving a  JSON response indicating: 
`"error":"Invalid check_id provided for upload"`

While trying to run these lines in Python 3.7:
`sdk = codequiry.Codequiry("MY_API_KEY")`
`upload = sdk.upload_file(14298, 'CodeToReview/PCTEST/TEST/0.zip')`
`print(upload)`

When I do a `.get_check()` with the same check_id I don't have any problems. The id is created correctly in the system.

Thanks for the help.

**Comment by mperry-uni** [comment]
I'm having exactly the same issue of invalid check_id for file_upload, I'm very keen to see how to solve it.

If I use the curl command in the documentation it is working, so it seems it is an issue with Python or the library, rather than the files I'm using.

https://codequiry.com/usage/api/upload-check

**Comment by christianledgard** [comment]
That’s correct @mperry-uni. I solved the problem changing the header of the request in the SDK.  There’s a issue with the “content-type”.

**Comment by mperry-uni** [comment]
What do I change the content-type to? Would very much appreciate if you can confirm how to fix this... it is causing me issues

**Comment by christianledgard** [comment]
When I send the request, I'm only posting the API Key on the header. That seems to solve the issue.

But now I'm facing another one. I can only check a small number (7-12) of submissions on a single check. When I upload a large check (27-30), It gets stuck on "Verifying files" and then a `"Check halted, took more than 3001 seconds to complete"` error. I communicate this to support, and they are investigating it.

**Comment by mperry-uni** [comment]
I ended up just writing a subprocess.run call to run the curl shell command. Good to know can do that in Python.
